src/views/HighlightView.vue

```diff
@@ -67,6 +73,26 @@ onMounted( updateProgress );
 	scroll-snap-align: start;
 }
 
+.wiki-highlight-view-topbar {
+	position: fixed;
+	top: 0;
+	left: 0;
+	right: 0;
+	height: 60px;
+	background-image: linear-gradient( to bottom, rgba( 0, 0, 0, 0.8 ), rgba( 0, 0, 0, 0 ) );
+}
+
+.wiki-highlight-view-topbar-icon {
+	display: inline-block;
+	padding: 10px;
+	width: 45px;
+	height: 45px;
+	background-image: url( ../assets/back-arrow-white.svg );
+	background-size: 25px;
+	background-position: center;
+	background-repeat: no-repeat;
+}
```

Comment:
the icon should stick to the app left most
![image](https://github.com/wikimedia/wiki-highlights/assets/2560096/b36b05a9-ff15-4952-b871-9b9b3696ef2f)

Reply:
I just pushed a fix for that, please try again